index.js

```diff
@@ -98,7 +105,13 @@ class ServerlessApiCloudFrontPlugin {
   }
 
   prepareOrigins(distributionConfig) {
-    distributionConfig.Origins[0].OriginPath = `/${this.options.stage}`;
+    const originPath = this.getConfig('originPath', `/${this.options.stage}`);
+
+    if (originPath !== null) {
```

Comment:
This comparison will always be true as `getConfig` returns default value in case of `originPath` missing in config.

I would suggest to get rid of it :)